src/script/components/InputBar/Components/ImageUploadButton/ImageUploadButton.tsx

```diff
@@ -22,15 +22,12 @@ import {useRef} from 'react';
 import {Icon} from 'Components/Icon';
 import {t} from 'Util/LocalizerUtil';
 
-import {Config} from '../../../Config';
-
```

Comment:
Generic components should not depend on something that is app-specific. 
The config should be injected by the consumer